### PR TITLE
Update course details

### PR DESCRIPTION
**Pull request by lmkeston**
updates for course pages

### CODE REVIEW

File: course-details.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
 
-In this course, you’ll learn how to:
+## What you'll learn
 
+We'll answer common questions like:
+- What is a pull request?
+- How do you comment on a pull request?
+- How do you review a pull request?
+- How do you request a pull request review?
+- How do you accept changes introduced by a pull request review?
```
Comment (hectorsector) [inline course-details.md]: I've been using this section as an opportunity to add details to things that can't be covered in a learning outcome (which is the section below), which are very specifically about the activities the learner completes. Scanning through the responses might help. For this course, for example, I'd maybe add some of the following questions:
- What are the responsibilities of a reviewer?
- What's a `CODEOWNERS` file?
- What's a `diff` and how do I use it? 
- How do I implement changes from a pull request review? 
- How can I suggest a change when reviewing a pull request?

File: course-details.md [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+- Create a pull request
 - Comment on pull requests
 - Leave reviews on pull requests
-- See what’s expected when reviewing pull requests
+- Know what’s expected when reviewing pull requests
 - Request pull request reviews
 - Accept changes introduced by a pull request review
 
-This course has a dedicated message board on the [GitHub Community Forum]({{ communityBoard }}). Create a post to start a conversation, discuss this course with GitHub Trainers and participants, or troubleshoot any issues you encounter. 
+
```
Comment (hectorsector) [inline course-details.md]: For the "What you'll build" section, I think it'd be nice to show a screenshot or GIF of the bot and the learner reviewing each other OR we could also show the final product, which is a Tetris game in this case.

File: course-details.md [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+## Prerequisites
+None. This course is a great introduction to using pull requests.
+
```
Comment (hectorsector) [inline course-details.md]: For the "Projects used" section, I think we can link to the source code for the Tetris game: https://github.com/jakesgordon/javascript-tetris

File: course-details.md [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
+## Audience
+
+New developers, new GitHub users, users new to Git, students, managers, teams
```
Comment (hectorsector) [inline course-details.md]: Looking great! I think this course is useful for "Developers" in general.

File: config.yml [hunk 5]
```diff
@@ -1,5 +1,4 @@
 title: Reviewing pull requests
-tagline: Improve your collaboration on GitHub with pull request reviews.
```
Comment (hectorsector) [inline config.yml]: 👍

File: config.yml [hunk 6]
```diff
@@ -1,5 +1,4 @@
...
 description: See how collaboration works on GitHub and start building great things, together.
 tags:
   - GitHub
```
Comment (hectorsector) [inline config.yml]: We can add up to 5 tags here!

File: course-details.md (hunk 2 above)
Comment (lmkeston) [reply course-details.md]: yeah, that's a good idea

File: course-details.md (hunk 1 above)
Comment (lmkeston) [reply course-details.md]: done

**Pull request merged**